### DISCUSSION TITLE
Using LAMBOptimizer with the torch in aragpt2?

### DISCUSSION THREAD

**Discussion by En-J-A** [aub-mind/arabert#122] [Q&A]
Can I use the optimizer (AdamW) of the gpt2 model as it is with aragpt2 instead of LAMBOptimizer ?
If not, How can I use LAMBOptimizer with the torch in aragpt2 because in your code, you use TensorFlow? Can you guide me, please?

**Comment by WissamAntoun** [comment]
yes of course, you can use any optimizer that you want it's fine.

**Reply by En-J-A** [reply]
Good. Thank you.
